jmix-quartz/quartz-flowui/src/main/java/io/jmix/quartzflowui/event/QuartzDataChangedEvent.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2024 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.jmix.quartzflowui.event;
+
+import com.vaadin.flow.component.Component;
+import org.springframework.context.ApplicationEvent;
+
+public class QuartzDataChangedEvent extends ApplicationEvent {
```

Comment:
Create events for job execution start and finish. Keep execution context and exception (if applicable) inside them (https://docs.jmix.io/jmix/quartz/index.html#track-job-execution)

Reply:
ok